### PR TITLE
add PeopleEatCheckbox, PeopleEatFavorite, PeopleEatShowPass, PeopleEatTabItem and PeopleEatCounter. Also update tailwind config

### PR DESCRIPTION
**Pull request by Llavrov**

### CODE REVIEW

File: src/components/standard/buttons/PeopleEatShowPass.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { styled } from '@mui/material';
+import Checkbox, { CheckboxProps } from '@mui/material/Checkbox';
```
Comment (CemYil03) [inline src/components/standard/buttons/PeopleEatShowPass.tsx:2]:
```suggestion
import Checkbox, { type CheckboxProps } from '@mui/material/Checkbox';
```

File: src/components/standard/buttons/PeopleEatFavorite.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { styled } from '@mui/material';
+import Checkbox, { CheckboxProps } from '@mui/material/Checkbox';
```
Comment (CemYil03) [inline src/components/standard/buttons/PeopleEatFavorite.tsx:2]:
```suggestion
import Checkbox, { type CheckboxProps } from '@mui/material/Checkbox';
```

File: src/components/standard/buttons/PeopleEatCheckbox.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import { styled } from '@mui/material';
+import Checkbox, { CheckboxProps } from '@mui/material/Checkbox';
```
Comment (CemYil03) [inline src/components/standard/buttons/PeopleEatCheckbox.tsx:2]:
```suggestion
import Checkbox, { type CheckboxProps } from '@mui/material/Checkbox';
```

File: .gitignore [hunk 4]
```diff
@@ -32,4 +32,4 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 
-.idea
+.idea/
```
Comment (CemYil03) [inline .gitignore:35]: This chamge is not required, you can leave it as is.
```suggestion
.idea
```

**Review by CemYil03 — CHANGES_REQUESTED** [review]
Mainly some things GitHub / eslint from verify suggested 💬

File: src/components/standard/buttons/PeopleEatNext.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            sx={{ border: 1, '$hover:': { backgroundColor: 'transparent' } }}
+            size={'small'}
+            variant="contained"
+            onClick={handleClick}
+            className={classNames('min-w-[46px] min-h-[46px] hover:bg-orange p-0 border-border hover:border-transparent', {
+                ['hover:bg-white']: !active,
+                ['bg-orange']: active,
+            })}
+        >
+            <PeopleEatIcon icon={Icon.arrowRight} Classes={classNames('', { ['invert']: active })} edgeLength={46} />
```
Comment (CemYil03) [inline src/components/standard/buttons/PeopleEatNext.tsx:30]: You can also delete the new Classes property on the PeopleEatIcon component. The suggestion below should achieve the same without the overhead.

```suggestion
            <PeopleEatIcon icon={Icon.arrowRight} className={classNames('', { ['invert']: active })} edgeLength={46} />
```

Comment (CemYil03) [inline src/components/standard/buttons/PeopleEatNext.tsx:30]: You can also delete the new Classes property on the PeopleEatIcon component. The suggestion below should achieve the same without the overhead.

```suggestion
            <PeopleEatIcon icon={Icon.arrowRight} className={classNames('', { ['invert']: active })} edgeLength={46} />
```

File: src/components/standard/icon/PeopleEatIcon.tsx [hunk 6]
```diff
@@ -2,9 +2,9 @@ import Image from 'next/image';
 import { type ReactElement } from 'react';
 import { type PeopleEatIconProps } from './PeopleEatIconProps';
 
-export default function PeopleEatIcon({ icon, edgeLength = 24 }: PeopleEatIconProps): ReactElement {
+export default function PeopleEatIcon({ icon, edgeLength = 24, Classes }: PeopleEatIconProps): ReactElement {
```
Comment (CemYil03) [inline src/components/standard/icon/PeopleEatIcon.tsx:5]:
```suggestion
export default function PeopleEatIcon({ icon, edgeLength = 24 }: PeopleEatIconProps): ReactElement {
```

File: src/components/standard/icon/PeopleEatIcon.tsx [hunk 7]
```diff
@@ -2,9 +2,9 @@ import Image from 'next/image';
...
     return (
-        <div>
+        <div className={Classes}>
```
Comment (CemYil03) [inline src/components/standard/icon/PeopleEatIcon.tsx:7]:
```suggestion
        <div>
```

File: src/components/standard/icon/PeopleEatIconProps.ts [hunk 8]
```diff
@@ -3,4 +3,5 @@ import { type Icon } from './Icon';
 export interface PeopleEatIconProps {
     icon: Icon;
     edgeLength?: number;
+    Classes?: string;
```
Comment (CemYil03) [inline src/components/standard/icon/PeopleEatIconProps.ts:6]:
```suggestion
    icon: Icon;
    edgeLength?: number;
```

**Review by CemYil03 — CHANGES_REQUESTED** [review]

File: src/components/standard/icon/PeopleEatIconProps.ts [hunk 9]
```diff
@@ -3,4 +3,5 @@ import { type Icon } from './Icon';
 export interface PeopleEatIconProps {
     icon: Icon;
     edgeLength?: number;
+    classes?: string;
```
Comment (CemYil03) [inline src/components/standard/icon/PeopleEatIconProps.ts:6]:
```suggestion
```

File: src/components/standard/icon/PeopleEatIcon.tsx [hunk 10]
```diff
@@ -2,9 +2,9 @@ import Image from 'next/image';
 import { type ReactElement } from 'react';
 import { type PeopleEatIconProps } from './PeopleEatIconProps';
 
-export default function PeopleEatIcon({ icon, edgeLength = 24 }: PeopleEatIconProps): ReactElement {
+export default function PeopleEatIcon({ icon, edgeLength = 24, classes }: PeopleEatIconProps): ReactElement {
     return (
-        <div>
+        <div className={classes}>
```
Comment (CemYil03) [inline src/components/standard/icon/PeopleEatIcon.tsx:7]:
```suggestion
        <div>
```

File: src/components/standard/icon/PeopleEatIcon.tsx [hunk 11]
```diff
@@ -2,9 +2,9 @@ import Image from 'next/image';
 import { type ReactElement } from 'react';
 import { type PeopleEatIconProps } from './PeopleEatIconProps';
 
-export default function PeopleEatIcon({ icon, edgeLength = 24 }: PeopleEatIconProps): ReactElement {
+export default function PeopleEatIcon({ icon, edgeLength = 24, classes }: PeopleEatIconProps): ReactElement {
```
Comment (CemYil03) [inline src/components/standard/icon/PeopleEatIcon.tsx:5]:
```suggestion
export default function PeopleEatIcon({ icon, edgeLength = 24 }: PeopleEatIconProps): ReactElement {
```

File: src/components/standard/buttons/PeopleEatNext.tsx [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+            sx={{ border: 1, '$hover:': { backgroundColor: 'transparent' } }}
+            size={'small'}
+            variant="contained"
+            onClick={handleClick}
+            className={classNames('min-w-[46px] min-h-[46px] hover:bg-orange p-0 border-border hover:border-transparent', {
+                ['hover:bg-white']: !active,
+                ['bg-orange']: active,
+            })}
+        >
+            <PeopleEatIcon icon={Icon.arrowRight} classes={classNames('', { ['invert']: active })} edgeLength={46} />
```
Comment (CemYil03) [inline src/components/standard/buttons/PeopleEatNext.tsx:30]:
```suggestion
            <PeopleEatIcon icon={Icon.arrowRight} className={classNames('', { ['invert']: active })} edgeLength={46} />
```

**Review by CemYil03 — CHANGES_REQUESTED** [review]

File: src/components/standard/icon/PeopleEatIcon.tsx (hunk 6 above)
Comment (Llavrov) [reply src/components/standard/icon/PeopleEatIcon.tsx]: classes - variable for additional actions with PeopleEatIcon like invertion in PeopleEatNext button

**Review by CemYil03 — APPROVED** [review]
Thank you for all the changes! 👏

**Pull request merged**